go/checkerlib/lib.go

```diff
@@ -190,22 +190,23 @@ func StoreState(key string, data interface{}) {
 	}
 }
 
-// LoadState allows to retrieve data stored through StoreState. If no data
-// exists for the given key (and the current service and team), nil is
-// returned.
-func LoadState(key string) interface{} {
+// LoadState allows to retrieve data stored through StoreState.
+// dataPtr must be a pointer for json.Unmarshal.
+// If no data exists for the given key (and the current service and team)
+// false is returned and the data the dataPtr is pointing to is unchanged.
+func LoadState(key string, dataPtr interface{}) bool {
```

Comment:
Please no line-breaks in the comment (it should be a single paragraph) and rename `dataPtr`.

```suggestion
// LoadState allows to retrieve data stored through StoreState by unmarshalling it into data.
// If no data exists for the given key (and the current service and team)
// false is returned.
func LoadState(key string, data interface{}) bool {
```